src/Conversions.cc

```diff
@@ -1424,6 +1424,38 @@ sdf::Sensor gazebo::convert(const msgs::Sensor &_in)
 
     out.SetCameraSensor(sensor);
   }
+
+  else if (out.Type() == sdf::SensorType::GPS ||
+           out.Type() == sdf::SensorType::NAVSAT)
+  {
+      sdf::NavSat sensor;
+      if (_in.has_gps())
+      {
+          if (_in.gps().position().has_horizontal_noise())
+          {
+              sensor.SetHorizontalPositionNoise(gazebo::convert<sdf::Noise>(_in.gps().position().horizontal_noise()));
```

Comment:
Wrap these lines (and the ones below) to 80 char, please

Reply:
done